src/Models/SchemaOrg/EngineSpecification.php

```diff
@@ -7,32 +7,40 @@
  */
 class EngineSpecification extends \OpenActive\Models\SchemaOrg\StructuredValue
 {
+    /**
+     * @return string[]|null
+     */
+    static public function getType()
```

Comment:
Added PR https://github.com/openactive/models-lib/pull/17 for PSR-2 PHP-FIG standards:
> static MUST be declared after the visibility.

Reply:
Will re-run the generator once your PR openactive/models-lib#17 is merged

Reply:
Just pushed the new version from fixed generator.